metrics/health/health.go

```diff
@@ -86,6 +86,7 @@ func (s httpMetricServer) HealthStatus(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
+	log.Info("Health check request", "chain", chainName, "url", r.URL.String(), "height", s.stats[chainName].Height , "lastUpdated", s.stats[chainName].LastUpdated)
```

Comment:
What is your goal?

Reply:
After a discussion with @ADobrodey we couldn't understand the reasons why Stats API replied with an HTTP timeout error. He said that it is a problem with the application. As far as the logs were empty it was decided to add `/health` requests logs to see if Stats API can write logs in general and see if `/health` requests reach the server at the time of the error because it wasn't clear what happened with the application at the moment of the error. 
Later I found in Grafana charts that the Stats API pod was updated and an HTTP timeout error happened at this time. Updated this information in DevOps ticket.

Reply:
I think right now it is up to you to decide if we need this PR request. From one side it is useful to see all logs. From the other side, there are too many logs in Bridge Relayer as `/health` requests happen very often.